Makefile

```diff
@@ -155,3 +156,19 @@ console:
 destroy: deploy-init
 	cd terraform && . workspace_variables/$(DEPLOY_ENV).sh \
 		&& terraform destroy -var-file=workspace_variables/$(DEPLOY_ENV).tfvars $(AUTO_APPROVE)
+
+enable-maintenance: ## make qa enable-maintenance / make prod enable-maintenance CONFIRM_PRODUCTION=y
+	$(if $(HOSTNAME), $(eval REAL_HOSTNAME=${HOSTNAME}), $(eval REAL_HOSTNAME=${APP_ENV}))
+	cf target -s ${SPACE}
+	cd service_unavailable_page && cf push
+	cf map-route find-unavailable find-postgraduate-teacher-training --hostname ${REAL_HOSTNAME}
```

Comment:
It should be the full domain without the prefix, ie: `find-postgraduate-teacher-training.service.gov.uk`